File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusProductVO.kt
package com.sdkwork.backend

data class PlusProductVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val userId: Int? = null,
    val title: String? = null,
    val code: String? = null,
    val subtitle: String? = null,
    val resources: AssetMediaResourceList? = null,
    val tags: TagsContent? = null,
    val price: Double? = null,
    val originalPrice: Double? = null,
    val stock: Int? = null,
    val salesCount: Int? = null,
    val status: String? = null,
    val onSaleAt: String? = null,
    val description: String? = null,
    val categoryId: Int? = null,
    val baseAttributes: Attributes? = null,
    val specAttributes: Attributes? = null
)

File: sdkwork-backend-sdk-csharp/Models/PlusProductVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusProductVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? UserId { get; set; }
        public string? Title { get; set; }
        public string? Code { get; set; }
        public string? Subtitle { get; set; }
        public AssetMediaResourceList? Resources { get; set; }
        public TagsContent? Tags { get; set; }
        public double? Price { get; set; }
        public double? OriginalPrice { get; set; }
        public int? Stock { get; set; }
        public int? SalesCount { get; set; }
        public string? Status { get; set; }
        public string? OnSaleAt { get; set; }
        public string? Description { get; set; }
        public int? CategoryId { get; set; }
        public Attributes? BaseAttributes { get; set; }
        public Attributes? SpecAttributes { get; set; }
    }
}
